# from django.core.exceptions import ValidationError


class AnimeValidator:
    def __init__(self): ...

    def __call__(self):
        return True
